titus-ext/aws/src/main/java/com/netflix/titus/ext/aws/appscale/AWSAppAutoScalingClient.java

```diff
@@ -241,14 +241,17 @@ public void onSuccess(DescribeScalableTargetsRequest request, DescribeScalableTa
 
             putScalingPolicyRequest.setTargetTrackingScalingPolicyConfiguration(targetTrackingConfigAws);
         } else {
-            return Observable.error(new UnsupportedOperationException("Scaling policy type " + policyConfiguration.getPolicyType().name() + " is not supported."));
+            return Observable.error(new UnsupportedOperationException(String.format(
+                    "Scaling policy type not supported: jobId=%s, policyId=%s, type=%s",
+                    jobId, policyRefId, policyConfiguration.getPolicyType().name()
+            )));
         }
 
         return RetryWrapper.wrapWithExponentialRetry(String.format("createOrUpdateScalingPolicy %s for job %s", policyRefId, jobId),
                 Observable.create(emitter -> awsAppAutoScalingClientAsync.putScalingPolicyAsync(putScalingPolicyRequest, new AsyncHandler<PutScalingPolicyRequest, PutScalingPolicyResult>() {
                     @Override
                     public void onError(Exception exception) {
-                        logger.error("Exception creating scaling policy ", exception);
+                        logger.error("Exception creating scaling policy: jobId={}, policyId={}", jobId, policyRefId, exception);
```

Comment:
I would call it policyRefId since policyId refers to AWS provided ID which doesn't exist in this scenario.